templates/mwe.md

```diff
@@ -21,26 +9,30 @@ If you need to link to Lean 3 specific resources please link there.
 
 ## tl;dr
 
-When posting code on Zulip, please include all `import`s and `open`s and `universe`s and `variable`s, so others can simply just cut and paste what you post, and see the same issue that you are seeing. The best way to ensure you have done this is to make a completely new Lean file containing only what you are proposing to post, and checking that it compiles.
+When posting code on Zulip, please include all `import`s, `open`s, `universe`s, and `variable`s, so others can simply just copy-paste what you post, and see the same issue that you are seeing.
+
+The best way to ensure you have done this is to copy-paste the code snipped you are about to post into the [lean web editor](https://lean.math.hhu.de) (or an empty Lean file) and check it compiles.
```

Comment:
How often is the lean web editor mathlib updated? Using it seems more complicated than using an empty Lean file, so I would use it as our first recommendation.

Reply:
It's currently updated every 6h, which we might change according to the demand, but it'll stay updated minimum once a day.

I thought that was less complicated, but I'm happy to have it changed around in the text.